CHANGELOG.md

```diff
@@ -27,6 +27,7 @@ For details about compatibility between different releases, see the **Commitment
 ### Fixed
 
 - When searching for end-devices, specifying `last_seen_at` as the field in which the devices will be sorted by no longer returns an error.
+- Event data not appearing in the Console.
```

Comment:
`v3.23.1` does not seem to have this behavior, so the changelog entry may be skipped.